dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -31,5 +31,6 @@ COPY ["supervisord.conf", "/etc/supervisor/conf.d/"]
 COPY ["ipinip.json.j2", "/usr/share/sonic/templates/"]
 COPY ["mirror.json.j2", "/usr/share/sonic/templates/"]
 COPY ["ports.json.j2", "/usr/share/sonic/templates/"]
+COPY ["msn27xx.32ports.buffers.json.j2", "/usr/share/sonic/templates/msn27xx.32ports.buffers.json.j2"]
```

Comment:
Destination filename not necessary, as we're not renaming the file. Please either remove it or add destination filenames to all copies for consistency.

Reply:
thanks. updated.